src/components/Calculator.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [OPTIONAL] This was not pointed out by the previous reviewer. As required by Microverse, you are not meant to use any linter disable rule in your codebase. The linters are there to make sure your code follows the best practices and to help you write better codes, you are not meant to disable the checks. I'll suggest you remove this and try to fix the issue. :small_airplane: :smile: :+1: